### PR TITLE
Update Datumaro version

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

- Updated Datumaro version
- Added Datumaro Skeleton annotation type

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat/apps/dataset_manager/bindings.py [hunk 1]
```diff
@@ -1075,18 +1075,17 @@ def _load_categories(labels: list):
         point_categories = datum_annotation.PointsCategories()
 
         for _, label in labels:
-            if label.get('parent') is None:
-                label_id = label_categories.add(label['name'])
-                for _, attr in label['attributes']:
-                    label_categories.attributes.add(attr['name'])
+            label_id = label_categories.add(label['name'], label.get('parent'))
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/bindings.py:1078]: Why logic in this place was changed?

File: cvat/apps/dataset_manager/bindings.py [hunk 2]
```diff
@@ -17,8 +17,8 @@
 import rq
 from attr import attrib, attrs
 from datumaro.components.dataset import Dataset
+from datumaro.components.media import ByteImage, Image, PointCloud
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/bindings.py:20]: Please check if using `import datumaro as dm` is reasonable here.

File: cvat/requirements/base.txt [hunk 3]
```diff
@@ -45,9 +45,7 @@ diskcache==5.0.2
 boto3==1.17.61
 azure-storage-blob==12.13.0
 google-cloud-storage==1.42.0
-# --no-binary=datumaro: workaround for pip to install
-# opencv-headless instead of regular opencv, to actually run setup script
-datumaro==0.2.0 --no-binary=datumaro
+git+https://github.com/cvat-ai/datumaro
```
Comment (zhiltsov-max) [inline cvat/requirements/base.txt:48]: This link leads to the develop branch. I don't think it is a good solution, better to specify a branch or commit.

Comment (yasakova-anastasia) [reply cvat/requirements/base.txt]: Fixed

File: cvat/apps/dataset_manager/bindings.py (hunk 2 above)
Comment (yasakova-anastasia) [reply cvat/apps/dataset_manager/bindings.py]: Updated. But it doesn't work for `PointCloud`.

**Review by kirill-sizov — APPROVED** [review]
LGTM

**Pull request merged**